app/collections/Products.coffee

```diff
@@ -13,3 +13,9 @@ module.exports = class Products extends CocoCollection
       return countrySpecificProduct
     else
       return @findWhere { name: 'basic_subscription' }
+
+  getLifetimeSubscriptionForUser: (user) ->
+    if countrySpecificProduct = @findWhere { name: "#{user?.get('country')}_lifetime_subscription" }
```

Comment:
Ah, I notice that's how it was before — well, now's a great time to improve that :)